Amazon.QLDB.Driver.IntegrationTests/AsyncStatementExecutionTests.cs

```diff
@@ -424,18 +244,7 @@ public async Task ExecuteAsync_InsertMultipleDocuments_DocumentsInserted()
 
             // When.
             var query = $"INSERT INTO {Constants.TableName} <<?,?>>";
-            var count = await qldbDriver.Execute(async txn =>
-            {
-                var result = await txn.Execute(query, parameters);
-
-                var count = 0;
-                await foreach (var row in result)
-                {
-                    count++;
-                }
-                return count;
-            });
-            Assert.AreEqual(2, count);
+            Assert.AreEqual(2, await ExecuteWithParamsAndReturnRowCount(query, parameters));
```

Comment:
The execute 5 lines below cannot use any of the new helper methods?

Reply:
Not really. The behaviour here is really specific and not repeated.